### PR TITLE
feat(matches): empty-state placeholder for homepage matches slider (#1323)

### LINKED ISSUE #1323: Homepage matches slider: uneven top/bottom padding + undefined empty state
## Summary

Two related issues on the `matches-slider` section of the homepage (`apps/web/src/app/page.tsx:260-266`, rendered via `MatchesSliderSection`):

1. **Visual padding asymmetry** — top and bottom padding of the dark section are not visually equal.
2. **Empty state (end of season)** — section disappears entirely when no matches are upcoming. Needs a designed placeholder. The section will sit in this state roughly 2-3 months per year (off-season), so it's a prominent surface on the homepage and deserves a proper design, not a fallback.

## Design + plan (approved)

- Design doc: [`docs/plans/2026-04-19-matches-slider-empty-state-design.md`](../blob/main/docs/plans/2026-04-19-matches-slider-empty-state-design.md)
- Implementation plan: [`docs/plans/2026-04-19-matches-slider-empty-state-plan.md`](../blob/main/docs/plans/2026-04-19-matches-slider-empty-state-plan.md)

### Empty-state placeholder (layered)

- **Baseline** (no editorial input): stadium/fans photo, motto *"Er is maar één plezante compagnie"*, secondary line *"We zijn terug in juli!"*, CTAs to PSD dashboard + Facebook.
- **Optional Sanity overlay** (`matchesSliderPlaceholder` object on home-page document):
  - `nextSeasonKickoff` (date) → server-computed countdown.
  - `announcementText` (+ optional `announcementHref`) → editorial message.
  - `highlightImage` (with required alt) → override default photo.
- **Decision rule**: countdown > announcement > motto baseline. Past/missing date falls back gracefully. All fields optional — no editorial work required to ship.
- **Two layout candidates in Storybook** (split 2:1, centered card) — pick by visual review, delete loser before merge.

### Padding asymmetry fix

Scoped `paddingBottom: "pb-16"` override on the matches-slider `SectionConfig` in `page.tsx`. No changes to `SectionStack` defaults or `SectionHeader.mb-10` (blast radius limited to this section).

### In-scope cleanup

- Extract social URLs to `EXTERNAL_LINKS` in `apps/web/src/lib/constants.ts` (currently hardcoded inline in `SocialLinks.tsx`).
- Add PSD dashboard URL as platform constant (`https://kcvv.prosoccerdata.com/dashboard`).
- **Remove Twitter everywhere**: `SocialLinks`, share buttons in `ArticleMetadata` and `PlayerShare`, `SITE_CONFIG.twitterHandle`, metadata, JSON-LD (`sameAs`), icons barrel, `llms.txt`, tests, stories.

## Acceptance criteria

- [ ] Top and bottom padding of the matches-slider section render visually equal relative to the diagonal transitions.
- [ ] Homepage with zero upcoming matches renders a coherent layout with the designed placeholder; Youth section's diagonal transition still looks correct (``prevBg=\"kcvv-black\"`` assumption preserved).
- [ ] `EXTERNAL_LINKS` owns all external URLs used by the placeholder + footer social icons; no inline hardcoded social URLs remain.
- [ ] Twitter fully removed: no references in `apps/web` source, tests, metadata, JSON-LD, or `SITE_CONFIG`.
- [ ] New Sanity field is optional — baseline state works without editorial input.
- [ ] Final layout choice shipped (losing layout deleted); Storybook covers the full variant matrix (baseline / countdown / announcement / countdown+announcement / custom image / all populated) + mobile snapshot.
- [ ] Unit tests cover the countdown decision rule and edge cases (past date, `n=0`, `n=1`, singular/plural).
- [ ] No hardcoded colors, no emoji (Lucide icons only).
- [ ] `pnpm --filter @kcvv/web check-all` passes; Storybook builds.

## Notes

- All copy is Dutch, brand voice (*"Er is maar één plezante compagnie"*, never *"meer dan een club"*).
- Default photo committed at `apps/web/public/images/home/matches-empty-state.jpg` (user supplies before Task 8 in the plan).
- Sanity migration: optional field, no data migration; staging studio tested first per project convention.

### PR DESCRIPTION
**Pull request by soniCaH**
Closes #1323

## Summary

- Designed empty-state placeholder for the homepage matches slider (shown ~2-3 months/year in off-season). Optional Sanity overlay with countdown / announcement / custom image; decision rule falls back to a motto baseline that requires no editorial input.
- Scoped `paddingBottom: "pb-16"` override on the matches-slider section to correct the top/bottom asymmetry (`SectionHeader.mb-10` biases the top spacing).
- Cleanup: Twitter removed everywhere (share buttons, `SITE_CONFIG.twitterHandle`, icon barrel, metadata, `llms.txt`); `EXTERNAL_LINKS` now owns PSD + social URLs.

## Deviations from the plan

- **Layout**: settled on the centered layout early (split discarded). Agreed during visual review after the initial split rendering looked awkward in its narrow column.
- **Photo**: reused existing `/images/youth-trainers.jpg` instead of downloading a new stock photo. The Sanity `highlightImage` override is the long-term answer; this is a stand-in.
- **Decoder shape**: Sanity placeholder flows through the existing `homepage.repository` pattern (typegen + VM) rather than a new Effect Schema file. Matches the existing Sanity/DDD boundary in this repo — Effect Schema is reserved for the PSD-facing `api-contract`.
- **Date math**: plain `Date.UTC(...)` + calendar-day diff rather than `date-fns` (already on Luxon for formatting; no new dep needed).

## Test plan

- [ ] Open Vercel preview homepage — matches are currently present, so slider should render normally; padding should read symmetric.
- [ ] In Sanity staging, edit the home-page document and set `matchesSliderPlaceholder.nextSeasonKickoff` to a date ~30 days out; verify countdown copy and ISR revalidation.
- [ ] Temporarily force `matches: []` locally (or wait for off-season) to eyeball the baseline empty state and verify the Youth diagonal transition still lands on `kcvv-black` correctly.
- [ ] Check both mobile and desktop.
- [ ] Storybook — `Features/Homepage/MatchesSliderEmptyState` has 9 variants covering baseline / countdown / today / announcement / combined / custom image / all fields.
- [ ] `pnpm --filter @kcvv/web check-all` passes locally (2435 tests).

## Follow-ups (not in this PR)

- Replace `/images/youth-trainers.jpg` with a proper single-frame stadium/atmosphere photo once one is available. Can be delivered via Sanity `highlightImage` without a redeploy.
- Visual review of `pb-16` override — may need to drop to `pb-14` or raise to `pb-18` once measured in-browser.
- Sanity Studio migration note: every new field is optional, no data migration required. Staging studio first per project convention.

🤖 Generated with [Claude Code](https://claude.com/claude-code)

### CODE REVIEW

**Comment by soniCaH** [comment]
@coderabbitai
fetchGroq uses Effect.orDie, so there's no typed HttpNotFound error; catchAll is a no-op here but consistent with neighbours.

File: apps/web/src/components/home/MatchesSliderEmptyState/MatchesSliderEmptyState.tsx [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+  children: React.ReactNode;
+  "aria-label"?: string;
+}) => (
+  <p
+    className="font-title text-3xl font-black leading-tight tracking-tight text-white md:text-5xl lg:text-6xl"
+    aria-label={ariaLabel}
+  >
+    {children}
+  </p>
+);
```
[the remark this answers is not in the document]
Comment (soniCaH) [reply apps/web/src/components/home/MatchesSliderEmptyState/MatchesSliderEmptyState.tsx:133]: SectionHeader already renders h2; making primary text h2 duplicates. Using h3 (correct outline)

**Comment by soniCaH** [comment]
@coderabbitai review
